charts/datahub/subcharts/acryl-datahub-actions/templates/deployment.yaml

```diff
@@ -159,6 +164,10 @@ spec:
                   name: {{ .Values.global.datahub.metadata_service_authentication.systemClientSecret.secretRef }}
                   key: {{ .Values.global.datahub.metadata_service_authentication.systemClientSecret.secretKey }}
             {{- end }}
+            {{- if or .Values.global.datahub.monitoring.enablePrometheus }}
+            - name: DATAHUB_ACTIONS_MONITORING_ENABLED
+              value: "true"
+            {{- end }}
```

Comment:
may we explicitly add `DATAHUB_ACTIONS_MONITORING_PORT` also?

Reply:
Yes, makes sense in case the default value in the image will ever be changed.